types/wlr_linux_dmabuf.c

```diff
@@ -207,15 +207,16 @@ static void params_create_common(struct wl_client *client,
 			// Skip checks if kernel does no support seek on buffer
 			continue;
 		}
-		if (buffer->attributes.offset[i] >= size) {
+		if (buffer->attributes.offset[i] > size) {
```

Comment:
The ```>=``` is here to make sure we have at least some data to read. Changing it to ```>``` means we're o.k. with having an offset but nothing left to read. I'd prefere to leave it as is but I'm o.k. either way.

Reply:
Hmm, the intention was to check here that the offset is okay, and in the next condition that the it's not empty.